nikola/data/themes/bootblog4-jinja/templates/base_helper.tmpl

```diff
@@ -64,7 +64,7 @@ lang="{{ lang }}">
 
 {% macro late_load_js() %}
     {% if use_cdn %}
-        <script src="http://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
```

Comment:
I’d prefer to stay with `code.jquery.com`, which supports https (and uses sha256, which is shorter and good enough).

```html
<script src="https://code.jquery.com/jquery-3.5.1.js" integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=" crossorigin="anonymous"></script>
```

Reply:
Sure, I'm happy to switch it back to `code.jquery.com`. When I manually calculated the sha256 hash though, I got the same value as was there previously with the insecure HTTP link. How did you calculate the hash to get `QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=`?

Reply:
I copied the wrong snippet (for the unminified version) from `code.jquery.org`. The correct one is:

```html
<script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
```

Reply:
Are there copy+pasteable script tags on `code.jquery.com`? I must be missing something.

Reply:
They’re not very discoverable, a modal box appears when you left-click one of `uncompressed, minified, slim, slim minified`:
![image](https://user-images.githubusercontent.com/327323/101999634-48b30f00-3cdf-11eb-99e7-d2f3bcd062ae.png)

Reply:
Wow. I've been to that page so many times and I always just right-click the links and select "Copy Link Location".